save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/UserSettingsView.kt

```diff
@@ -44,6 +43,8 @@ import kotlinx.coroutines.launch
 import kotlinx.serialization.encodeToString
 import kotlinx.serialization.json.Json
 
+val orderedOrganizationStatus = listOf(CREATED, DELETED, BANNED).withIndex().associate { it.value to it.index }
```

Comment:
Enum entries already have their own order: the order in which they're declared in the enum class.

Why can't we just use `enumValues<OrganizationStatus>()` here?

Reply:
I used Order, so I got rid of this